src/board/board.mjs

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable no-use-before-define */
+import { checkForWin } from './game';
+
+import {
+  duplicateGrid,
+  shuffleGrid,
+  flattenGrid,
+  flatToMultiDimensional,
+} from './helpers';
+
+import {
+  removeElementByClass,
+  removeChildren,
+} from './dom-helpers';
+
+let flatAnswerKey = null;
+let currentFlatGrid = null;
+let gameStarted = false;
+let initialClick = null;
+
+function resetGame() {
+  flatAnswerKey = null;
+}
+
+function addPlayButton(grid) {
+  // removes any existing play buttons or winning messages before adding button
+  removeElementByClass('.play-button');
```

Comment:
based on your comments on line 26, maybe a good name for this function would be `clearBoard` or `resetBoard` instead of `removeElementByClass`, then you wouldnt need that comment